### PR TITLE
[Release] 2.15.3

### PR DESCRIPTION
**Pull request by abelgardep**
Implements https://github.com/owncloud/android/issues/2963
Needs https://github.com/owncloud/android-library/pull/346

### CODE REVIEW

**Comment by jesmrec** [comment]
## QA

- [X] Test issues inside https://github.com/owncloud/android/milestone/45
- [X] Execute tests: https://github.com/owncloud/android-scenario-testing/tree/master/src/test/resources/io/cucumber
- [ ] Check notifications:
  - [ ] Download single file
  - [ ] Download multiple files
  - [ ] Upload single file 
  - [ ] Upload multiple files
  - [ ] Error in download
  - [ ] Error in upload
  - [ ] Conflict
- [ ] Upgrade from 2.15.2

**Comment by jesmrec** [comment]
Approved on my side

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 2.15.3 (October 2020)
+- Bug fixes, including:
+  + Fix a crash related with downloads notifications
```
Comment (theScrabi) [inline CHANGELOG.md:3]:
```suggestion
  + Fix a crash related to downloads notifications
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+  + Potential fix to ANR when retrying camera uploads
```
Comment (theScrabi) [inline CHANGELOG.md:4]:
```suggestion
  + Potential fix for ANR when retrying camera uploads
```

**Review by theScrabi — CHANGES_REQUESTED** [review]

**Review by theScrabi — APPROVED** [review]

**Pull request merged**